shell/src/main/java/tachyon/shell/command/CatCommand.java

```diff
@@ -37,7 +36,7 @@
    * @param conf the configuration for Tachyon
    * @param tfs the filesystem of Tachyon
    */
-  public CatCommand(TachyonConf conf, TachyonFileSystem tfs) {
+  public CatCommand(TachyonConf conf, FileSystem tfs) {
```

Comment:
seems like it belongs in this PR since we rename tfs to fs elsewhere, and the type is changing, but I can wait if you want to get this in ASAP